src/nvim/mbyte.c

```diff
@@ -1092,9 +1090,52 @@ bool utf_iscomposing(int c)
   return intable(combining, ARRAY_SIZE(combining), c);
 }
 
+#ifdef __SSE2__
+
+# include <emmintrin.h>
+
+// Return true for characters that can be displayed in a normal way.
+// Only for characters of 0x100 and above!
+bool utf_printable(int c)
+  FUNC_ATTR_CONST
+{
+  if (c == 0x70f) {
+    return false;
+  }
+  if (c < 0x180b || c > 0xffff) {
+    return true;
+  }
```

Comment:
Will this be faster as smaller codepoints are more common?
```suggestion
  if (c < 0x180b || c > 0xffff) {
    return true;
  }
  if (c == 0x70f) {
    return false;
  }
```
Also codepoints above `0xffff` are not that common, so maybe removing the `c > 0xffff` condition here (as the SIMD comparisons below also cover it) will be faster?